src/tasks/installers/olm.ts

```diff
@@ -313,43 +318,53 @@ export class OLMTasks {
           task.title = `${task.title}: ${ctx.isPreInstalledOLM}...OK`
         },
       },
+      {
+        title: `Check if operator is installed in ${DEFAULT_OPENSHIFT_OPERATORS_NS_NAME} namespace`,
```

Comment:
Why do we need this?

Reply:
I didn't found a better way to get the ctx.operatorNamespace. I will dig